main.nf

```diff
@@ -83,38 +71,23 @@ if (params.help){
 
 // Configurable variables
 params.name = false
-<<<<<<< HEAD
-// params.fasta = params.genome ? params.genomes[ params.genome ].fasta ?: false : false
-=======
-params.fasta = params.genome ? params.genomes[ params.genome ].fasta ?: false : false
->>>>>>> TEMPLATE
 params.multiqc_config = "$baseDir/conf/multiqc_config.yaml"
 params.email = false
 params.plaintext_email = false
 
 multiqc_config = file(params.multiqc_config)
```

Comment:
You might want to put these files into channels and use the files from the channels...

Reply:
See here for an example: https://github.com/nf-core/eager/pull/113/files

Reply:
`file()` creates a channel doesn't it? if so `mutliqc_config` and `output_docs` have their channels line 78-79.

Note that you didn't comment on the latest version here, I haven't had a `params.fasta` for a long time

Reply:
I'm not sure this creates a channel, unfortunately. We had some issues with e.g. files like the multiqc_config not being staged properly when using just the `file()` method...

Reply:
done!